.scrutinizer.yml

```diff
@@ -0,0 +1,23 @@
+filter:
```

Comment:
I feel like this should be excluded in favour of travis, unless there are open source options for Scrutinizer?